TextInputKit/TextInputKit/Code/TextInputFormat/TextInputFormat+Formatter.swift

```diff
@@ -98,6 +115,17 @@ private final class FormatterAdapter<Value> : Formatter {
             .sameRange(in: originalString.utf16)
             .sameRange(in: originalString)
 
+        let originalSelectedRange: Range<String.Index> = {
+            #if os(macOS)
+            if options.tracksCurrentEditorSelection {
```

Comment:
As I see, there are two options:
- As it's implemented now;
- To assign a closure, which will provide the selected range, to the `Formatter`.

Do you have any ideas?

I think that, choosing from the two options, the implemented variant is more straightforward.